src/poc/server/poc-server/package.json

```diff
@@ -15,11 +15,16 @@
     "body-parser": "^1.20.3",
     "cors": "^2.8.5",
     "dotenv": "^16.4.5",
+<<<<<<< HEAD
+    "ejs": "^3.1.10",
```

Comment:
ejs can be removed from the package.json

Reply:
removed